app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+  def index
+    # @users = User.all
+  end
+
+  def show
+    # @user = User.find(params[:id])
+  end
+
+  def posts
+    # @posts = Post.where(user_id: params[:id])
+  end
+
+  def post
+    # @post = Post
+    #   .where(user_id: params[:id])
+    #   .where(id: params[:id])
+    #   .first
+  end
```

Comment:
- [ ] Great work so far 👍 but it would be nice if you add these method in the deidcated controller for posts 👍 because you have user controller to maintain all the pages of users so there should be a controller for posts to handle all the posts views/pages 😊

**Hint: Your posts method should be index and the post method should be show method in the posts_controller.rb 👍**